src/components/Player/Buttons/ButtonsComponent.html

```diff
@@ -0,0 +1,13 @@
+<audio class="mediaPlayer__music" src="<%= src %>"></audio>
```

Comment:
please fix file and class naming according to BEM
please add wrapper div with name of BEM block